generator/src/main/java/org/apache/kafka/message/MessageSpecType.java

```diff
@@ -27,5 +27,8 @@
     RESPONSE,
 
     @JsonProperty("header")
-    HEADER;
+    HEADER,
+
+    @JsonProperty("data")
+    DATA;
```

Comment:
Just a small addition to denote that the class we're generating is going to be used as "userdata", not a header, nor a request/response pair.